src/plugins/intel_npu/src/al/include/npu.hpp

```diff
@@ -27,6 +27,10 @@ class IEngineBackend : public std::enable_shared_from_this<IEngineBackend> {
     virtual const std::shared_ptr<IDevice> getDevice(const ov::AnyMap& paramMap) const;
     /** @brief Provide a list of names of all devices, with which user can work directly */
     virtual const std::vector<std::string> getDeviceNames() const;
+    /** @brief Provide driver version */
+    virtual const uint32_t getDriverVersion() const;
```

Comment:
Can you consider removing `const`?
```suggestion
    virtual uint32_t getDriverVersion() const;
```

I believe you cannot enforce user of the method to make it const for them, so we can save some space here :)

Reply:
good catch, thank you! done